feat(specs): upgrade PrecastConcrete category properties

OlehChyhyryn opened the pull request:
asana: [Updates to precast category](https://app.asana.com/0/698742722983597/1207326942987195/f)

alexlitvinenko reviewed (commented):
1. @alexche-cchange , would not "element_type" be more conventional name for a property then "precast_element_type"?
2. Improve commit message to something like: "add `element_type` property to `Precast` spec".

OlehChyhyryn commented:
> 1. @alexche-cchange , would not "element_type" be more conventional name for a property then "precast_element_type"?
> 2. Improve commit message to something like: "add `element_type` property to `Precast` spec".

I will add it as a description, because there is several changes.

1. Added new category CivilPrecast to OpenEpd specification
2. Added precast_element_type properties to each of PrecastCategory subcategories
